### ISSUE TITLE
Change New Note Creation Behavior to match with the Mobile App

### ISSUE THREAD

**Issue by christianhans** [standardnotes/web#388]
Hi,

I really do not like the new behavior when creating notes. It is inconsistent with the mobile App as well.

Previously (before v3.3?) when clicking "+" the cursor directly went into the Title field.

Now it takes me into the Note field and pre-populates the title with "Note 123" for example. I get the idea, but "Note 123" does help me to identify the note at all so I have to click Shift+Tab two times to delete "Note 123" so I can just leave the title empty (as the mobile App already does it).

I suggest changing the Web/Desktop App's behavior to match with the Mobile App. I.e. when creating a note the cursor should jump directly into the Note field and the Title should be left blank. Notes without titles are perfectly fine visualized both in Desktop and Mobile Apps. And I think putting the cursor into the Note field instead of Title field and leaving the Title blank is much better for taking quick notes. What's really cumbersome right now is having to delete "Note 123" all the time (since it doesnt help me and it adds extra unused information to my quick notes).

Best,
Christian

**Comment by mobitar** [comment]
Yeah, after spending some time with the new behavior, I also liked it focusing on the title on creation. Focusing on note body also doesn't always work with external editors since they don't all autofocus.

**Comment by JaspalSuri** [comment]
Thank you for your feedback; I've kept a note of it for the team to reference.
